WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteCredentialsViewController.swift

```diff
@@ -515,7 +556,7 @@ extension SiteCredentialsViewController {
     override func displayRemoteError(_ error: Error) {
         configureViewLoading(false)
         let err = error as NSError
-        if err.code == 403 {
+        if err.code == 403 || err.code == 401 {
```

Comment:
Nit: Would it help to add a comment to explain these error codes? Or moving into a constants enum would be even better. 😅

Reply:
Actually I think it makes more sense to let the host app decide whether to display the wrong credential alert. Updated in d094005.